programs/survival/portalorient.sc

```diff
@@ -0,0 +1,186 @@
+__config() ->
+(
+	m(
+		l('scope','player'),
+		l('stay_loaded',true),
+		
+		l('commands',
+			m(
+				l('<mode>','__change')
+			)
+		),

```

Comment:
Add a command with no argumetns that prints info on the three modes and maybe alos what the app does exactly? Since apps are now meant to be downloaded directly from in-game, we souldn't rely on people reading the code to understand the apps.

Reply:
Gave it a first attempt, please comment on the language in case anything is unclear. It's effectively a condensed copy of the docs description.